vpdq/sample-hashes/README.md

```diff
@@ -0,0 +1,4 @@
+# Summary
+These txt hash files are generated from VPDQ. These are used for regression test to check if the hashes created locally match with the original implementation.
```

Comment:
nit: from 'internal' VPDQ implementation.

Reply:
I am using the external VPDQ hashes because Barrett suggests that no one that uses the OSS version has the access to the internal implementation.